### PR TITLE
Remove and supress errors

### LINKED ISSUE #61: Fix QA errors/warnings
Our IG has quite a few errors and warnings: https://build.fhir.org/ig/SvenskaIndustriProfiler/fhir/qa.min.html and they are valid issues that require addressing. Once they are fixed, applying good discipline to never introduce new errors or warnings - in my experience - makes it much easier to stay on top of them.

I have filed a similar issue on [HL7 Sweden repo](https://github.com/HL7Sweden/basprofiler-r4/issues/56).

### PR DESCRIPTION
**Pull request by peturv**
Changes to address errors and warnings in our profiles.
Relaxing value -> pattern
Restricting extension context to Organization

Added simple examples
Added ignore lines for unavoidable & inherited errors

### CODE REVIEW

**Comment by niclas86** [comment]
@peturv please solve the conflict

**Comment by niclas86** [comment]
@vadi2 @MangalaEkanayake please review this

File: input/fsh/OrganizationSECommonsLite.fsh [hunk 1]
```diff
@@ -84,4 +84,12 @@ Description: """This is a light weight Swedish Organization profile.
 
 * address.line.extension contains
             iso21090-ADXP-careOf named careOf 0..* MS
-        and iso21090-ADXP-streetAddressLine named streetAddressLine 0..* MS
\ No newline at end of file
+        and iso21090-ADXP-streetAddressLine named streetAddressLine 0..* MS
+
+Instance: OrganizationSELiteExample
```
Comment (MangalaEkanayake) [inline input/fsh/OrganizationSECommonsLite.fsh]: Which style we'll follow? I preper 2nd option :-)
1. Have all the examples in the profile definition fsh file?
2. maintain examples separately in separate files in ./input/fsh/example folder?

Comment (vadi2) [reply input/fsh/OrganizationSECommonsLite.fsh]: I have a preference for the first, but it's not a very strong one.

**Review by vadi2 — COMMENTED** [review]
The branch does not build unfortunately:

```
error InstanceOf OrganizationSEVendorLite not found for OrganizationSELiteExample
  File: /Users/vadim.peretokin/Programs/fhir/input/fsh/OrganizationSECommonsLite.fsh
  Line: 89 - 95
error InstanceOf PractitionerSEVendorLite not found for PractitionerSELiteExample
  File: /Users/vadim.peretokin/Programs/fhir/input/fsh/PractitionerSECommonsLite.fsh
  Line: 24 - 31
```

File: input/fsh/OrganizationSECommonsLite.fsh (hunk 1 above)
Comment (niclas86) [reply input/fsh/OrganizationSECommonsLite.fsh]: If the second one is being used more wide I okay with that,

**Comment by peturv** [comment]
Awesome pipeline you have added @vadi2 ! I'll see what I can merge from it to the publication repository.

**Review by vadi2 — APPROVED** [review]

**Pull request merged**